scripts/bash_aliases.sh

```diff
@@ -0,0 +1,50 @@
+alias nav-bag-down='mkdir -p ~/bags/"`date +%Y-%m-%d`" && cd ~/bags/"`date +%Y-%m-%d`" && rosbag record /down/left/camera_info /down/left/image_raw /odom /absodom /clock'
+alias nav-bag-velodyne='mkdir -p ~/bags/"`date +%Y-%m-%d`" && cd ~/bags/"`date +%Y-%m-%d`" && rosbag record /velodyne_points /odom /absodom /clock'
+alias nav-bag-sick='mkdir -p ~/bags/"`date +%Y-%m-%d`" && cd ~/bags/"`date +%Y-%m-%d`" && rosbag record /scan /odom /absodom /clock'
+alias nav-bag-velodyne-stereo='mkdir -p ~/bags/"`date +%Y-%m-%d`" && cd ~/bags/"`date +%Y-%m-%d`" && rosbag record /velodyne_points /stereo/left/camera_info /stereo/left/image_raw /stereo/right/camera_info /stereo/right/image_raw /odom /absodom /clock'
+alias nav-bag-sick-stereo='mkdir -p ~/bags/"`date +%Y-%m-%d`" && cd ~/bags/"`date +%Y-%m-%d`" && rosbag record /scan /stereo/left/camera_info /stereo/left/image_raw /stereo/right/camera_info /stereo/right/image_raw /odom /absodom /clock'
+alias nav-bag-velodyne-right='mkdir -p ~/bags/"`date +%Y-%m-%d`" && cd ~/bags/"`date +%Y-%m-%d`" && rosbag record /velodyne_points /right/right/camera_info /right/right/image_raw /odom /absodom /clock'
+alias nav-bag-sick-right='mkdir -p ~/bags/"`date +%Y-%m-%d`" && cd ~/bags/"`date +%Y-%m-%d`" && rosbag record /scan /right/right/camera_info /right/right/image_raw /odom /absodom /clock'
+alias nav-bag-motors='mkdir -p ~/bags/"`date +%Y-%m-%d`" && cd ~/bags/"`date +%Y-%m-%d`" && rosbag record /c3_trajectory_generator/waypoint /trajectory /wrench/cmd /BL_motor/cmd /BL_motor/feedback /BL_motor/status /BR_motor/cmd /BR_motor/feedback /BR_motor/status /FL_motor/cmd /FL_motor/feedback /FL_motor/status /FR_motor/cmd /FR_motor/feedback /FR_motor/status /odom /absodom /clock'
+alias nav-bag-status='mkdir -p ~/bags/"`date +%Y-%m-%d`" && cd ~/bags/"`date +%Y-%m-%d`" && rosbag record /c3_trajectory_generator/waypoint /trajectory /wrench/cmd /battery_monitor /adaptation /diagnostics /learn /alarm /alarm_raise /odom /absodom /clock'
+alias nav-bag-hydrophones='mkdir -p ~/bags/"`date +%Y-%m-%d`" && cd ~/bags/"`date +%Y-%m-%d`" && rosbag record /hydrophones/debug /hydrophones/ping /hydrophones/pose /hydrophones/processed /velodyne_points /odom /absodom /clock'
+alias nav-bag-hydrophones-stereo='mkdir -p ~/bags/"`date +%Y-%m-%d`" && cd ~/bags/"`date +%Y-%m-%d`" && rosbag record /stereo/left/camera_info /stereo/left/image_raw /stereo/right/camera_info /stereo/right/image_raw /hydrophones/debug /hydrophones/ping /hydrophones/pose /hydrophones/processed /velodyne_points /odom /absodom /clock'
+alias nav-bag-lqrrt='mkdir -p ~/bags/"`date +%Y-%m-%d`" && cd ~/bags/"`date +%Y-%m-%d`" && rosbag record /stereo/left/camera_info /ogrid /unclassified_markers /stereo/left/image_raw /velodyne_points /odom /absodom /lqrrt/effor /lqrrt/focus /lqrrt/goal /lqrrt/impact /lqrrt/path /lqrrt/ref /lqrrt/tree /move_to/feedback /move_to/goal /move_to/result /clock'
```

Comment:
It doesn't seem scalable to pollute the environment with this many bagging aliases and variables. Consider enclosing this bag stuff in a function `set_bag_env`.